cmd/nerdctl/image_convert.go

```diff
@@ -67,6 +68,16 @@ func newImageConvertCommand() *cobra.Command {
 	imageConvertCommand.Flags().Bool("zstdchunked", false, "Use zstd compression instead of gzip (a.k.a zstd:chunked). Should be used in conjunction with '--oci'")
 	// #endregion
 
+	// #region nydus flags
+	imageConvertCommand.Flags().Bool("nydus", false, "Convert an OCI image to Nydus image. Should be used in conjunction with '--oci'")
+	imageConvertCommand.Flags().String("nydus-builder-path", "nydus-image", "The nydus-image binary path, if unset, search in PATH environment")
+	imageConvertCommand.Flags().String("nydus-work-dir", "/tmp", "Work directory path for image conversion")
+	imageConvertCommand.Flags().String("nydus-chunk-dict", "", "Specify the path of the chunk dictionary")
+	imageConvertCommand.Flags().String("nydus-prefetch-patterns", "", "The file path pattern list want to prefetch")
+	imageConvertCommand.Flags().String("nydus-compressor", "lz4_block", "The algorithm to compress image data blob, default: lz4_block, possible values: [none, lz4_block, gzip, zstd]")
+	imageConvertCommand.Flags().String("nydus-fs-version", "5", "Version number of nydus image format, possible values: 5, 6")
```

Comment:
I don't think nydus image can be compressed as gzip...

Reply:
But I got the algorithms supported [here](https://github.com/dragonflyoss/image-service/blob/b0c3588facdddc07a446c1ae4411fa02f6a1a2e5/src/bin/nydus-image/main.rs#L225-L232).

Reply:
I am afraid that nydus can't run on gizp format since nydusd is assuming gzip is only for stargz format.  Perhaps we should remove gzip from nydus-image help message
Moreover, it is not productive ready yet even it can be used

Reply:
OK. Then I will remove the `gzip` option from the help message. :)